presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/api/ConfigAPI.java

```diff
@@ -41,12 +41,12 @@
 @Singleton
 public final class ConfigAPI
 {
-    private final APIFileHandler apiFileHandler;
+    private final ApiFileHandler apiFileHandler;
```

Comment:
I only did it here because it's at the beginning of the name, where it's a lot worse to have an acronym.

I let the other ones because the correct name for those is really `Resource`, as in Presto, and that sort of widespread change is much more likely to cause problems than this one.